src/Twig/Extension/SonataDoctrineORMAdminExtension.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Twig\Extension;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class SonataDoctrineORMAdminExtension extends AbstractExtension
+{
+    public function getFunctions()
+    {
+        return [
+            new TwigFunction('deprecate_template', [$this, 'deprecateTemplate']),
+        ];
+    }
+
+    public function deprecateTemplate($templateName)
+    {
+        @trigger_error(
+            "The $templateName template is deprecated since version 3.x and will be removed in 4.0."
```

Comment:
Please quote `$templateName`

Reply:
And the line seems to be to long

Reply:
hm, will add quotes and try to linebreak it.